framework/sandstone_tests.h

```diff
@@ -95,6 +102,9 @@ class SandstoneTestSet
     std::vector<struct test_cfg_info> enable(const char *name);
     struct test_cfg_info enable(struct test *t);
 
+    std::vector<struct test_cfg_info> enable_by_feature(uint64_t enabled_features_mask);
+    std::vector<struct test_cfg_info> disable_by_feature(uint64_t disabled_features_mask);
+
```

Comment:
These calls (API) are not needed. `SandstoneTestSet` is instantiated once after arguments are processed. Just pass the values from the options in `test_set_cfg` and change `enable()`/`disable()` to take it into account.

Reply:
Fixed